client/http/http.go

```diff
@@ -171,7 +159,7 @@ func (h *httpClient) Get(ctx context.Context, round uint64) (Result, error) {
 	}
 	defer randResponse.Body.Close()
 
-	randResp := RandomData{}
+	randResp := client.RandomData{}
 	if err := json.NewDecoder(randResponse.Body).Decode(&randResp); err != nil {
 		return nil, err
```

Comment:
A note for the future (not criticism of this PR): In our error handling code (especially in prioritizing client), we never distinguish between network errors and client response errors (like invalid JSON or like "insufficient response" below).

Reply:
That's another one that needs to be recorded as an issue, rather than fixed here.